config/manifests/kustomization.yaml

```diff
@@ -1,8 +1,23 @@
 # These resources constitute the fully configured set of manifests
 # used to generate the 'manifests/' directory in a bundle.
 resources:
-- bases/codeflare-operator.clusterserviceversion.yaml
 - ../default
 - ../scorecard
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+
+configMapGenerator:
+- name: codeflare-stack-config
+  envs:
+    - bases/params.env
```

Comment:
I'd suggest we take that opportunity to move these two files in the `manifests` directory and remove the `bases` one. It's a leftover from the operator SDK for how it handles the CSV file, and it's not relevant anymore.